gadget/gadget.go

```diff
@@ -110,6 +110,17 @@ func (vs *VolumeStructure) IsBare() bool {
 	return vs.Filesystem == "none" || vs.Filesystem == ""
 }
 
+// EffectiveRole returns the role of given structure
+func (vs *VolumeStructure) EffectiveRole() string {
+	if vs.Role != "" {
+		return vs.Role
+	}
+	if vs.Role == "" && vs.Type == "mbr" {
+		return "mbr"
```

Comment:
A high-level wondering: would it make sense to have roles defined via consts in a central place, or it is not worth it? Is it just "gpt" and "mbr"?

Reply:
Consts added